apps/buildproof_dapp/app/utils/submit-hackathon/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const VALUE_PER_ATOM = BigInt("1000000000000000"); // 0.001 ETH par atome
```

Comment:
Is that the minimum deposit / creation fees ? 

if that's the case, it should be dynamic, we can either : 
- put in into the .env as  those values will be different between the testnet and mainnet
- ideally : get this value from the directly from the ETHMultiVault contract